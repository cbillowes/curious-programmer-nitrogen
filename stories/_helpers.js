import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page