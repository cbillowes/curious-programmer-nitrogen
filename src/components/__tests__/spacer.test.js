import React from 'react'
import Spacer from '../spacer'
import { getSnapshot } from './_helpers'

describe.skip(`Spacer`, () => {
  it(`should render`, () => {
    const spacer = (
      <Spacer />
    )
    const tree = getSnapshot(spacer)
    expect(tree).toMatchSnapshot()
  })
})
