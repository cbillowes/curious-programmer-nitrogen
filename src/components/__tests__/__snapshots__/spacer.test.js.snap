// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should render default 1`] = `
<div
  className="spacer no-heading"
/>
`;

exports[`Spacer should render large 1`] = `
<div
  className="spacer large"
/>
`;

exports[`Spacer should render small 1`] = `
<div
  className="spacer small"
/>
`;
